pages/productos/crear.tsx

```diff
@@ -17,7 +18,14 @@ const Create: NextPage = () => {
   const [createProduct, { loading, error: mutationError }] = useMutation(CREATE_PRODUCT)
 
   const submitHandler = async (newProduct: ProductEditable) => {
-    createProduct({ variables: { ...newProduct, providerId: isProvider } })
+    await createProduct({ variables: { ...newProduct, providerId: isProvider } })
+    Swal.fire({
+      title: 'Producto creado',
+      icon: 'success',
+      showConfirmButton: false,
+      timer: 3000,
+      timerProgressBar: true,
+    })
     push('/productos')
```

Comment:
What about if the request fails?

Reply:
Will be solved in another issue, as this one is mostly about message